src/datamodule.py

```diff
@@ -60,7 +61,14 @@ def read_chip(self, chip_path):
 
         # read timestep & normalize
         date = chip.tags()["date"]  # YYYY-MM-DD
-        year, month, day = self.normalize_timestamp(date)
+        (
+            year,
+            month,
+            day,
+            year_non_norm,
+            month_non_norm,
+            day_non_norm,
+        ) = self.normalize_timestamp(date)
```

Comment:
We are already returning the non-normalized date as a YYYY-MM-DD string no? Instead of using `ts.append(batch["timestep_non_norm"])` in in your partial-inputs-flood-tutorial.ipynb notebook, could you use `ts.append(batch["date"])`?

Reply:
Fixed this. It worked, thanks!